src/lightstep-datasource/partials/config.html

```diff
@@ -70,7 +73,10 @@ <h3 class="page-heading">HTTP</h3>
       <input class="gf-form-input" type="text"
              ng-model='ctrl.current.jsonData.dashboardURL' placeholder="https://app.lightstep.com"
              bs-typeahead="getSuggestUrls"  min-length="0"
-             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"></input>
+             ng-pattern="/^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/"
+             ng-init='if (!ctrl.current.jsonData.dashboardUR) { ctrl.current.url = "https://app.lightstep.com" }'
```

Comment:
is `dashboardUR` a typo?

Reply:
Should this update `dashboardURL` instead of `url`?

Reply:
fixed.